Nos/Views/Moderation/ContentFlagView.swift

```diff
@@ -23,11 +23,13 @@ struct ContentFlagView: View {
                         title: String(localized: .localizable.flagContentSendTitle),
                         subtitle: nil
                     )
+                    .transition(.move(edge: .leading).combined(with: .opacity))
                 }
             }
         }
         .padding()
         .background(Color.appBg)
+        .animation(.easeInOut, value: selectedFlagOptionCategory)
```

Comment:
I'm seeing a build error on this line. Are you seeing it, too?

Reply:
> I'm seeing a build error on this line. Are you seeing it, too?

Oh my God! Thanks for catching this @joshuatbrown. I removed the Equatable conformity while merging the branch with the base branch. I have pushed the fix.